src/pages/profile/Profile.module.css

```diff
@@ -1,17 +1,39 @@
-button.btn_editProfile {
-  border: thin solid var(--COLOR-BACKGROUND-DARK-100);
+button.btn {
```

Comment:
suggestion: avoid over qualifying the selector, I see in many places, if its really required then fine otherwise ignore